packages/amplication-client/src/models.ts

```diff
@@ -1891,6 +1897,7 @@ export type ServiceSettings = IBlock & {
   serverSettings: ServerSettings;
   updatedAt: Scalars['DateTime'];
   versionNumber: Scalars['Float'];
+  dbType?: Scalars['String'];
```

Comment:
Did you add it manually?

Reply:
Yes

Reply:
Our models are generated automatically. I noticed that you added it to the DTO on the server. Remove it from here and run `npx nx build amplication-server`. You will see how the models are being changed automatically